lessons/git/intro/git-init.md

```diff
@@ -0,0 +1,11 @@
+# Installing and running Git and GitHub
```

Comment:
Just realized this file should not even be there. This just confuses everybody. I am sticking to slides-2017 and practice-code-2017